cibuildwheel/windows.py

```diff
@@ -19,7 +19,7 @@
 IS_RUNNING_ON_TRAVIS = os.environ.get('TRAVIS_OS_NAME') == 'windows'
 
 
-def shell(args, env=None, cwd=None):
+def call(args, env=None, cwd=None):
```

Comment:
`call()` on macOS doesn't use a shell by default, here Windows always does. I think that's why the names are different. If we're changing it, I'd recommend using a shell only where necessary - when running user commands.

Reply:
It might actually be nice to keep the distinction, then?

Reply:
it is not possible because of this bug https://bugs.python.org/issue8557 (with `shell=False` environment variable PATH passed with `env=env` is ignored). 

Change name allow to use same build function for both systems.

Reply:
Ah, that is a problem. Probably that means we should stick with `shell`, for now.

Reply:
Ok. I revert this change.